sources/packages/web/src/components/generic/TitleValue.vue

```diff
@@ -14,13 +16,13 @@ Used when we need to display title and the value inside a container
 import { defineComponent } from "vue";
 export default defineComponent({
   props: {
-    propertyTitle: {
+    propertyValue: {
       type: String,
       required: true,
     },
-    propertyValue: {
+    propertyTitle: {
```

Comment:
why this change?

Reply:
In props propertyTitle was moved after propertyValue because propertyTitle is not a required property anymore because of the slot provided to create title.

Reply:
an order doesn't matter here right?